test/e2e/proxy_test.go

```diff
@@ -640,13 +641,13 @@ func testProxyServiceSessionAffinity(ipFamily *corev1.IPFamily, ingressIPs []str
 	tableSessionAffinityName := "SessionAffinity"
 	tableSessionAffinityOutput, _, err := data.RunCommandFromPod(metav1.NamespaceSystem, agentName, "antrea-agent", []string{"ovs-ofctl", "dump-flows", defaultBridgeName, fmt.Sprintf("table=%s", tableSessionAffinityName)})
 	require.NoError(t, err)
-	if *ipFamily == corev1.IPv4Protocol {
+	if slices.Contains(ipFamilies, corev1.IPv4Protocol) {
 		require.Contains(t, tableSessionAffinityOutput, fmt.Sprintf("nw_dst=%s,tp_dst=80", svc.Spec.ClusterIP))
 		require.Contains(t, tableSessionAffinityOutput, fmt.Sprintf("load:0x%s->NXM_NX_REG3[]", strings.TrimLeft(hex.EncodeToString(nginxIP.ipv4.To4()), "0")))
 		for _, ingressIP := range ingressIPs {
 			require.Contains(t, tableSessionAffinityOutput, fmt.Sprintf("nw_dst=%s,tp_dst=80", ingressIP))
 		}
-	} else {
+	} else if slices.Contains(ipFamilies, corev1.IPv6Protocol) {
```

Comment:
should this really be an `else`? I feel like with this change, we should be able to support the dual-stack case, and both sets of checks should be validated.

Of course, we need to be careful with `svc.Spec.ClusterIP`. Instead, we should use the appropriate address from `svc.Spec.ClusterIPs`.

Reply:
As we discussed offline, I think we only support the dual-stack case for flow-visibility e2e test currently.